cypress/e2e/apps.spec.js

```diff
@@ -51,7 +51,7 @@ describe('The apps', function() {
 
 		it('Renders the default files list', function() {
 			cy.login('jane', { route: 'apps/files' })
-			cy.get('.files-fileList tr').should('contain', 'welcome.txt')
+			cy.get('.files-fileList tr, files-list__tbody tr').should('contain', 'welcome.txt')
```

Comment:
Shouldn't it be a class selector? 

```suggestion
			cy.get('.files-fileList tr, .files-list__tbody tr').should('contain', 'welcome.txt')
```

Reply:
Good point. I wonder why the test in question passed never the less but failed on last run in main.